backend/lib/testing/mock-gateways/mock-mongo.repository.ts

```diff
@@ -147,6 +147,10 @@ export class MockMongoRepository
     throw new Error('Method not implemented.');
   }
 
+  async getLegacyCaseNotesPage(..._ignore): Promise<any> {
+    throw new Error('getMalformedCaseNotesPage not yet implemented');
```

Comment:
The rest of the file follows this pattern:
```suggestion
    throw new Error('Method not implemented.');
```